static/src/javascripts/bower.json

```diff
@@ -36,7 +36,7 @@
       "lodash-amd": "bower_components/lodash-amd/compat/**",
       "domready": "bower_components/domready/ready.js",
       "when": "bower_components/when/es6-shim/Promise.js",
-      "video.js": "bower_components/video.js/dist/video.min.js",
+      "video.js": "bower_components/video.js/dist/video.js",
```

Comment:
Out of curiosity, how come we changed this? Makes it easier to debug, that's for sure—and it's still minised later by Uglify through us :-)